client/state/selectors/get-current-user-payment-methods.js

```diff
@@ -32,6 +31,8 @@ const paymentMethods = {
 		BE: [ 'credit-card', 'bancontact', 'paypal' ],
 		CN: [ 'credit-card', 'alipay', 'paypal' ],
 		DE: [ 'credit-card', 'giropay', 'paypal' ],
+		// TODO: @ramonjd check this entry with @yoavf
+		IN: [ 'credit-card', 'emergent-paywall' ],
```

Comment:
We should add 'paypal' as well - we currently disable Paypal for India in INR transactions, but it's enabled for USD or other currencies transactions.